src/logic/safe/store/reducer/safe.ts

```diff
@@ -51,6 +51,8 @@ const updateSafeProps = (prevSafe, safe) => {
           List.isList(safe[key])
             ? record.set(key, safe[key])
             : record.update(key, (current) => current.merge(safe[key]))
+        } else {
+          record.set(key, safe[key])
```

Comment:
This took me a while to grasp. Please add a comment here.

Reply:
Please merge `dev` into this branch as it no longer has the hotfix.

Reply:
done!